assets/xml/objects/object_snowwd.xml

```diff
@@ -2,8 +2,8 @@
     <File Name="object_snowwd" Segment="6">
         <DList Name="object_snowwd_DL_000190" Offset="0x190" />
         <DList Name="object_snowwd_DL_000198" Offset="0x198" />
-        <!-- <Blob Name="object_snowwd_Blob_0002A0" Size="0x1000" Offset="0x2A0" /> -->
+        <Texture Name="object_snowwd_Tex_001AA0" OutName="tex_001AA0" Format="rgba16" Width="32" Height="64" Offset="0x1AA0" />
         <Texture Name="object_snowwd_Tex_0012A0" OutName="tex_0012A0" Format="rgba16" Width="32" Height="32" Offset="0x12A0" />
-        <!-- <Blob Name="object_snowwd_Blob_001AA0" Size="0x1000" Offset="0x1AA0" /> -->
+        <Texture Name="object_snowwd_Tex_002A0" OutName="tex_002A0" Format="rgba16" Width="32" Height="64" Offset="0x2A0" />
```

Comment:
Somehow, the order of `object_snowwd_Tex_001AA0` and `object_snowwd_Tex_002A0` got swapped. Could you fix it so that it's in increasing offset order?